<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-06T15:47:55 IST" name="Default suite" finished-at="2024-06-06T15:49:44 IST" duration-ms="108642">
    <groups>
    </groups>
    <test started-at="2024-06-06T15:47:55 IST" name="Default test" finished-at="2024-06-06T15:49:44 IST" duration-ms="108642">
      <class name="test.SwagLabLoginTest">
        <test-method is-config="true" signature="openSwagLab()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:47:55 IST" name="openSwagLab" finished-at="2024-06-06T15:48:49 IST" duration-ms="53418" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openSwagLab -->
        <test-method signature="loginWithLockedOutUser()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:48:49 IST" name="loginWithLockedOutUser" finished-at="2024-06-06T15:48:55 IST" duration-ms="6777" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at test.SwagLabLoginTest.loginWithLockedOutUser(SwagLabLoginTest.java:75)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithLockedOutUser -->
        <test-method is-config="true" signature="closeSwagLab()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:48:56 IST" name="closeSwagLab" finished-at="2024-06-06T15:48:56 IST" duration-ms="402" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeSwagLab -->
        <test-method is-config="true" signature="openSwagLab()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:48:56 IST" name="openSwagLab" finished-at="2024-06-06T15:49:08 IST" duration-ms="11930" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openSwagLab -->
        <test-method signature="loginWithPerformanceGlitchUser()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:49:08 IST" name="loginWithPerformanceGlitchUser" finished-at="2024-06-06T15:49:16 IST" duration-ms="8173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithPerformanceGlitchUser -->
        <test-method is-config="true" signature="closeSwagLab()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:49:16 IST" name="closeSwagLab" finished-at="2024-06-06T15:49:16 IST" duration-ms="148" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeSwagLab -->
        <test-method is-config="true" signature="openSwagLab()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:49:16 IST" name="openSwagLab" finished-at="2024-06-06T15:49:25 IST" duration-ms="8366" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openSwagLab -->
        <test-method signature="loginWithProblemUser()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:49:25 IST" name="loginWithProblemUser" finished-at="2024-06-06T15:49:27 IST" duration-ms="2712" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithProblemUser -->
        <test-method is-config="true" signature="closeSwagLab()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:49:28 IST" name="closeSwagLab" finished-at="2024-06-06T15:49:29 IST" duration-ms="1548" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeSwagLab -->
        <test-method is-config="true" signature="openSwagLab()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:49:29 IST" name="openSwagLab" finished-at="2024-06-06T15:49:41 IST" duration-ms="11566" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openSwagLab -->
        <test-method signature="loginWithValidUser()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:49:41 IST" name="loginWithValidUser" finished-at="2024-06-06T15:49:43 IST" duration-ms="2582" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at test.SwagLabLoginTest.loginWithValidUser(SwagLabLoginTest.java:59)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithValidUser -->
        <test-method is-config="true" signature="closeSwagLab()[pri:0, instance:test.SwagLabLoginTest@51399530]" started-at="2024-06-06T15:49:44 IST" name="closeSwagLab" finished-at="2024-06-06T15:49:44 IST" duration-ms="229" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeSwagLab -->
      </class> <!-- test.SwagLabLoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
